client/my-sites/site-settings/form-writing.jsx

```diff
@@ -116,7 +116,7 @@ class SiteSettingsFormWriting extends Component {
 					isSaving={ isSavingSettings }
 					onButtonClick={ handleSubmitForm }
 					showButton
-					title={ translate( 'Content Types' ) }
+					title={ translate( 'Content types' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Content Types' )` *ES Score: 9*